Dockerfile

```diff
@@ -18,9 +18,10 @@ RUN apt-get update && \
         xfonts-75dpi xfonts-base xfonts-scalable xvfb git ninja-build curl ccache clang libpython3-dev \
         libqt53dcore5 libqt53dextras5 libqt53dlogic5 libqt53dinput5 libqt53drender5 qt3d5-dev \
         qt3d-assimpsceneimport-plugin qt3d-defaultgeometryloader-plugin qt3d-gltfsceneio-plugin \
-        qt3d-scene2d-plugin libqt5serialport5-dev libexiv2-dev grass-dev && \
+        qt3d-scene2d-plugin libqt5serialport5-dev libexiv2-dev grass-dev binutils && \
     apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* \
```

Comment:
missing "&&" before the "\\" and the next command ?

Reply:
+1 :-)